apps/style/common.scss

```diff
@@ -809,15 +828,43 @@ $aniGifPreviewHeight: 60px;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
+
+  #show-toolbox-icon, #hide-toolbox-icon {
+    position: absolute;
```

Comment:
These get absolutely positioned so they don't interfere with centering of the header text.